lib/nostrum/struct/message.ex

```diff
@@ -4,12 +4,12 @@ defmodule Nostrum.Struct.Message do
   """
 
   alias Nostrum.Struct.Embed
-  alias Nostrum.Struct.User
   alias Nostrum.Struct.Message.Activity
   alias Nostrum.Struct.Message.Application
   alias Nostrum.Struct.Message.Attachment
   alias Nostrum.Struct.Message.Reaction
   alias Nostrum.Struct.Snowflake
+  alias Nostrum.Struct.User
```

Comment:
Small nitpick, but everywhere else we group similar alias together, this and some of the other struct modules being an exception. I think we should go ahead and change that while we're in here. If you feel like it you can go through and change it for any other offending modules, but if you don't then just this module would be great!

Reply:
Do you mean something like the following:

```
Nostrum.Struct.{Embed, Snowflake, User}
Nostrum.Struct.Message.{Activity, Application, Attachment, Reaction}
```
Or

```
Nostrum.Struct.{Embed, Message.Activity, Message.Application, Message.Attachment, Message.Reaction, Snowflake, User}
```

Not sure if the last form is even possible.

Reply:
I was implying the former. IMO the latter looks really messy.